### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by nohwnd**

### CODE REVIEW

**Comment by nohwnd** [comment]
Not my fault this is stale. @bishal-pdMSFT could you merge it please?

**Comment by bishal-pdMSFT** [comment]
Really sorry @nohwnd , I was not tracking this 😞 
can you please update the task's patch verison in task.json (and task.loc.json). Then I will merge.

**Comment by nohwnd** [comment]
done

**Review by anatolybolshakov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
@nohwnd thanks for the fix!

**Review by max-zaytsev — APPROVED** [review]

File: Tasks/UseDotNetV2/task.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 184,
-        "Patch": 0
+        "Patch": 1
```
Comment (anatolybolshakov) [inline Tasks/UseDotNetV2/task.json:17]: Could you please remove version bump? There are changes only for md file - so it won't affect task.

**Review by anatolybolshakov — CHANGES_REQUESTED** [review]
Could you please take a look at the comments?

**Pull request closed without merging**